src/main/java/org/broadinstitute/hellbender/tools/exome/CopyNumberTriStateAllele.java

```diff
@@ -0,0 +1,120 @@
+package org.broadinstitute.hellbender.tools.exome;
+
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.vcf.VCFHeader;
+import htsjdk.variant.vcf.VCFSimpleHeaderLine;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.hmm.CopyNumberTriState;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Enum of possible CNV alleles.
+ */
+@SuppressWarnings("serial")
+public class CopyNumberTriStateAllele extends Allele {
+
+    public static final CopyNumberTriStateAllele REF = new CopyNumberTriStateAllele(CopyNumberTriState.NEUTRAL, "N", true);
```

Comment:
I disagree. NEUT is certainly not clear (is not even a word nor a recognizable word-shortcut). NEUTRAL looks awful when contrasted with the other two 3 letter alleles DEL and DUP. It would be just better to change them all to NEUTRAL, DELETION and DUPLICATION. I wonder whether i could simply merge CopyNumberTriState and CopyNumberTriStateAllele... the problem is that it must be an Allele subclass and an enum makes more sense (more elegant) to represent a hidden state.

So my offer:

1. Leave as it is.
2. Change REF, DEL and DUP to NEUTRAL, DELETION, DUPLICATION.
3. Do 2 an substitute CopyNumberTriState into CopyNumberTriStateAllele (the name of the remaining class could be CopyNumberTriState but in anycase it won't an enum any longer.

@davidbenjamin please choose.